components/Card/Card.tsx

```diff
@@ -71,6 +78,16 @@ export const Card = (CardProps: CardProps) => {
       {<CardListItems items={shown} />}
       {bodyText && (
         <CardBootstrap.Body>
+          {bodyImage && (
```

Comment:
I like your suggestion to add a `body?: ReactElement` prop that we show instead of CardBootstrap.Body and below if present, otherwise show bodyText. Then you can pass in the img from the parent, and also omit the style props